pkg/ams/terms.go

```diff
@@ -0,0 +1,43 @@
+package ams
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/redhat-developer/app-services-cli/internal/build"
+)
+
+var fallbackTocSpec = []byte(`{
+    "kafka":{
+       "EventCode":"register",
+       "SiteCode":"ocm",
+       "StopOnTermsChange": true
+    },
+    "service-registry":{
+       "EventCode":"onlineService",
+       "SiteCode":"ocm",
+       "StopOnTermsChange": true
+    }
+  }
+`)
+
+func GetRemoteTermsSpec() TermsAndConditionsSpec {
+	response, err := http.Get(build.TermsReviewSpecURL)
+
+	var specJson []byte
+	if err != nil {
```

Comment:
I was considering 3 use cases when we get a network error:

1) Fail entire command with a message: Error: downloading terms specification failed
2) Print error but let users continue with creating service - accepting that they would 
3) Provide failover for download by having spec in the CLI. 

I have done some pros and cons and went with option nr3.

Nr1 will be too internal for end-users to understand and result in a very bad experience. There is no way we can troubleshoot this etc.

Nr2 will be the same and also will require handling of the error from backend but without terms URL.